src/site/content/en/blog/isinputpending/index.md

```diff
@@ -0,0 +1,194 @@
+    // Yield if we have to handle an input event, or we're out of time.
+    setTimeout(processWorkQueue);
+    return;
+  }
+  let job = workQueue.shift();
+  job.execute();
+}
+```
+
+By introducing a call to `navigator.scheduling.isInputPending()`, we're able to
+respond to input quicker while still ensuring that our display-blocking work
+executes uninterrupted otherwise. If we're not interested in handling anything
+other than input (e.g. painting) until work is complete, we can handily increase
+the length of `QUANTUM` as well.  
+  
+By default, "continuous" events are not returned from `isInputPending()`. These
+include `mousemove`, `pointermove`, and others. If you're interested in yielding for
+these as well, no problem. By providing a dictionary to `isInputPending()` with
+`includeContinuous` set to `true`, we're good to go:
+
+```js
+const DEADLINE = performance.now() + QUANTUM;
+const options = { includeContinuous: true };
+while (workQueue.length > 0) {
+  if (navigator.scheduling.isInputPending(options) || performance.now() >= DEADLINE) {
+    // Yield if we have to handle an input event (any of them!), or we're out of time.
+    setTimeout(processWorkQueue);
+    return;
+  }
+  let job = workQueue.shift();
+  job.execute();
+}
+```
+
+That's it! Frameworks like React are building `isInputPending()` support into their
+core scheduling libraries using similar logic. Hopefully, this will lead
+developers who use these frameworks to be able to benefit from `isInputPending()`
+behind the scenes without significant rewrites.
+
+## Yielding isn't always bad
+
+It's worth noting that yielding less isn't the right solution for every use
+case. There are many reasons to return control to the browser other than to
+process input events, such as to perform rendering and execute other scripts on
+the page.
+
+There exist cases where the browser isn't able to properly attribute pending
+input events. In particular, setting complex clips and masks for cross-origin
+iframes may report false negatives (i.e. `isInputPending()` may unexpectedly return
+false when targeting these frames). Be sure that you're yielding often enough if
+your site does require interactions with stylized subframes.
+
+Be mindful of other pages that share an event loop, as well. On platforms such
+as Chrome for Android, it's quite common for multiple origins to share an event
+loop. `isInputPending()` will never return `true` if input is dispatched to a
+cross-origin frame, and thus backgrounded pages may interfere with the
+responsiveness of foreground pages. You may wish to reduce, postpone, or yield
+more often when doing work in the background using the [Page Visibility
+API](https://www.w3.org/TR/page-visibility-2/).
+
+We encourage you to use `isInputPending()` with discretion. If there isn't
+user-blocking work to be done, then be kind to others on the event loop by
+yielding more frequently. [Long tasks can be
+harmful](/long-tasks-devtools/).
+
+## Conclusion
```

Comment:
```suggestion
harmful](/long-tasks-devtools/).

## Feedback

* TODO

## Conclusion
```

Can developers contact you two on Twitter? Does the spec have a repo?

Reply:
Yes to all of the above! Twitter links above, the repo can be found at https://github.com/WICG/is-input-pending.